### ISSUE TITLE
Viewport-sensitive thumbnail lazy loading

### ISSUE THREAD

**Issue by bakape** [bakape/meguca#300]
Reimplement thumbnail lazy loading with checks for what is inside the viewport. Client-side renders don't need this. Instead of individual per-model listeners, it would be more efficient to have a centralised algorithm. 

Depends on #281

**Comment by bakape** [comment]
- Should use the existing `client/render` functionality much like `scheduleFlush()` and execute within the same animation frame as other deferred DOM access.

**Comment by bakape** [comment]
No longer planned with current server-side rendering.
